phinx.yml

```diff
@@ -0,0 +1,32 @@
+paths:
```

Comment:
Rename this to `phinx.yml.dist` so that it can be overloaded with a custom version.